### PR TITLE
Update the "Tested up to" after testing in WordPress 6.2

### PR DESCRIPTION
**Pull request by renatho**
### Changes proposed in this Pull Request

* It just updates the "Tested up to" after testing in WordPress 6.2.
  * I'm updating it for parity, but I'm also updating it directly in the published plugin code without a release.

### CODE REVIEW

**Review by fjorgemota — APPROVED** [review]

File: readme.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ Contributors: Viper007Bond, automattic, donncha
 Donate link: https://alex.blog/2019/03/13/in-memory-of-alex-donation-link-update/
 Tags: code, source, sourcecode, php, syntax highlighting, syntax, highlight, highlighting, highlighter, WordPress.com
 Requires at least: 5.7
```
Comment (fjorgemota) [inline readme.txt:5]: Shouldn't we bump this too? :eyes:

File: syntaxhighlighter.php [hunk 2]
```diff
@@ -12,7 +12,7 @@
 License:      GPL2
 License URI:  https://www.gnu.org/licenses/gpl-2.0.html
 Requires at least: 5.7
```
Comment (fjorgemota) [inline syntaxhighlighter.php:14]: Shouldn't we bump this too? :eyes:

**Review by fjorgemota — APPROVED** [review]
Actually, just left a question, but the change per se is still approved.

File: readme.txt (hunk 1 above)
Comment (renatho) [reply readme.txt:5]: I think for this case that we're not introducing new features, we don't need to bump it. And we make sure that it's a change with less noise for the users using it too. :)

**Pull request merged**